docs/resources/artifactory_user.md

```diff
@@ -25,7 +22,7 @@ The following arguments are supported:
 
 * `name` - (Required) Username for user.
 * `email` - (Required) Email for user.
-* `password` - (Optional) Password for the user. When omitted, a random password is generated according to default Artifactory password policy.
+* `password` - (Optional) Password for the user. When omitted, a random password is generated according to current Artifactory password policy.
```

Comment:
That's impossible to know. The password is generated based on the default, out of the box. Not current